src/App.js

```diff
@@ -0,0 +1,36 @@
+const { once } = require('events');
+const path = require('path');
+const express = require('express');
+const config = require('./config.js').config();
+const debug = require('debug');
+
+const info = debug('info');
+const error = debug('error');
+
+class App {
+    constructor({ staticPath = '../resources/html' } = {}) {
```

Comment:
Forgot to mention it during the first review: we have the dedicated `config` for default values.